composables/useViewport.ts

```diff
@@ -1,7 +1,10 @@
 import { useWindowSize } from '@vueuse/core'
 
 export default function () {
-  const isMobile = computed(() => useWindowSize().width.value <= 768)
+  const { width } = useWindowSize()
+  const isMobile = computed(() => width.value <= 768)
+  const isDesktop = computed(() => width.value >= 1024)
+  const isFullHD = computed(() => width.value >= 1440)
 
-  return { isMobile }
+  return { isMobile, isDesktop, isFullHD }
```

Comment:
updated to use css helpers to toggle visibility. I still have to keep the code in `useViewport` because I need to use `isFullHD` to switch the `.container.is-fluid` class.

Reply:
> I still have to keep the code in `useViewport` because I need to use `isFullHD` to switch the `.container.is-fluid` class.

wdym? just to add `is-fluid`? which will be remove in #9279, in that cas I would remove `container` class if not used like a container

Reply:
it was used like this:

```vue
<div class="container" :class="{ 'is-fluid': isFullHD }">
```

But I've figured out how to get the same results:

`relative w-full mx-auto px-[1.25rem] md:px-[2.5rem] min-[1440px]:max-w-[1440px]`